flyteidl/protos/flyteidl/core/execution.proto

```diff
@@ -73,6 +73,8 @@ message ExecutionError {
         SYSTEM = 2;
     }
     ErrorKind kind = 4;
+    // Worker that generated the error
+    string worker = 5;
```

Comment:
Why no timestamp?

Reply:
Sure, we can add that. This type is used for the resolved error. The RFC called for including the worker, I think there is no harm in adding the timestamp as well.

Reply:
Addressed.